### PR TITLE
Add build script

### PR DESCRIPTION
**Pull request by mattdrees**
This PR adds a simple build script that can be used from Jenkins.

The new jenkins build is here: http://jenkins.uscm.org/job/syslog-splunk/

### CODE REVIEW

**Comment by mattdrees** [comment]
@twinge would you review for me?

File: build.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/ruby -w
+
+push = !ARGV.delete('--push').nil? || !ARGV.delete('-p').nil?
```
Comment (twinge) [inline build.rb]: `push = ARGV.delete('--push') || ARGV.delete('-p')`

Comment (mattdrees) [inline build.rb]: thanks; fixed

**Comment by twinge** [comment]
:+1:

**Comment by mattdrees** [comment]
thanks!

**Pull request merged**